### ISSUE TITLE
[Feature Request] CORS on AWS AppSync GraphQL calls

### ISSUE THREAD

**Issue by neilferreira** [aws/aws-appsync-community#277]
It would be great if AWS AppSync was able to support CORS in a similar fashion to the AWS API Gateway and other API services.

As it stands, there is no ability to restrict the allowed hostnames to limit the use of our API.

**Comment by fedecastelli** [comment]
yeah, it would be great otherwise anyone can create a fake public website and use your API. I wonder why they didn't implement it from the beginning.

**Comment by lorienpatt** [comment]
Any updates on this?  Seems strange they have not implemented this.

**Comment by Murtaza-arif** [comment]
Any updates on this? Not sure what is the reason behind not adding the cors option in appsync?

**Comment by mikkilevon** [comment]
Updates please?

**Comment by BadrBelhiti** [comment]
Any updates please?

**Comment by andylim0221** [comment]
Update?

**Comment by theharunyilmaz** [comment]
Any updates? 👀

**Comment by hunganhAtWhill** [comment]
Bump

**Comment by lpatterson-humanetics** [comment]
Any update on this?
